src/test/java/net/sf/jabref/logic/importer/fetcher/AstrophysicsDataSystemTest.java

```diff
@@ -57,6 +60,88 @@ public void setUp() throws Exception {
+        famaeyMcGaughEntry.setField("author", "Famaey, B. and McGaugh, S. S.");
+        famaeyMcGaughEntry.setField("title", "Modified Newtonian Dynamics (MOND): Observational Phenomenology and Relativistic Extensions");
+        famaeyMcGaughEntry.setField("journal", "Living Reviews in Relativity");
+        famaeyMcGaughEntry.setField("year", "2012");
+        famaeyMcGaughEntry.setField("volume", "15");
+        famaeyMcGaughEntry.setField("month", "#sep#");
+        famaeyMcGaughEntry.setField("archiveprefix", "arXiv");
+        famaeyMcGaughEntry.setField("doi", "10.12942/lrr-2012-10");
+        famaeyMcGaughEntry.setField("eprint", "1112.3960");
+        famaeyMcGaughEntry.setField("keywords", "astronomical observations, Newtonian limit, equations of motion, extragalactic astronomy, cosmology, theories of gravity, fundamental physics, astrophysics");
+        famaeyMcGaughEntry.setField("abstract", "A wealth of astronomical data indicate the presence of mass" + NEWLINE
+                + "discrepancies in the Universe. The motions observed in a variety of" + NEWLINE
+                + "classes of extragalactic systems exceed what can be explained by the" + NEWLINE
+                + "mass visible in stars and gas. Either (i) there is a vast amount of" + NEWLINE
+                + "unseen mass in some novel form - dark matter - or (ii) the data indicate" + NEWLINE
+                + "a breakdown of our understanding of dynamics on the relevant scales, or" + NEWLINE
+                + "(iii) both. Here, we first review a few outstanding challenges for the" + NEWLINE
+                + "dark matter interpretation of mass discrepancies in galaxies, purely" + NEWLINE
+                + "based on observations and independently of any alternative theoretical" + NEWLINE
+                + "framework. We then show that many of these puzzling observations are" + NEWLINE
+                + "predicted by one single relation - Milgrom's law - involving an" + NEWLINE
+                + "acceleration constant a\\_0 (or a characteristic surface density" + NEWLINE
+                + "{$\\Sigma$}\\_{\\dagger} = a\\_0/G) on the order of the square-root of the" + NEWLINE
+                + "cosmological constant in natural units. This relation can at present" + NEWLINE
+                + "most easily be interpreted as the effect of a single universal force law" + NEWLINE
+                + "resulting from a modification of Newtonian dynamics (MOND) on galactic" + NEWLINE
+                + "scales. We exhaustively review the current observational successes and" + NEWLINE
+                + "problems of this alternative paradigm at all astrophysical scales, and" + NEWLINE
+                + "summarize the various theoretical attempts (TeVeS, GEA, BIMOND, and" + NEWLINE
+                + "others) made to effectively embed this modification of Newtonian" + NEWLINE
+                + "dynamics within a relativistic theory of gravity."+  NEWLINE);
```

Comment:
Abstracts are copyright protected... Whether you can remove it or you have to take an other entry.